src/types.ts

```diff
@@ -101,3 +111,15 @@ export interface Column {
   stylingInfo: string[];
   sortDirection: string;
 }
+
+export interface RenderWithCarbonArguments {
+  env: Galaxy;
+  rootElement: HTMLElement;
+  model: EngineAPI.IGenericObject | undefined;
```

Comment:
Could this be optional instead of `| undefined`?

Reply:
Yes